gms/client/src/main/java/com/linkedin/dataprocess/client/DataProcesses.java

```diff
@@ -93,19 +100,37 @@ private DataProcessUrn getUrnFromKey(@Nonnull ComplexResourceKey<DataProcessKey,
   @Override
   public CollectionResponse<DataProcess> search(@Nonnull String input, @Nullable Map<String, String> requestFilters,
       @Nullable SortCriterion sortCriterion, int start, int count) throws RemoteInvocationException {
-    throw new UnsupportedOperationException(
-        String.format("%s doesn't support search feature yet,",
-            this.getClass().getName())
-    );
+    DataProcessesFindBySearchRequestBuilder requestBuilder = DATA_PROCESSES_REQUEST_BUILDERS
+        .findBySearch()
+        .inputParam(input)
+        .sortParam(sortCriterion)
+        .paginate(start, count);
+    if (requestFilters != null) {
```

Comment:
you don't need to check for null condition, `newFilter` will take care of it (https://github.com/linkedin/datahub/blob/master/metadata-dao/src/main/java/com/linkedin/metadata/dao/utils/QueryUtils.java#L40)
Here is an e.g. https://github.com/linkedin/datahub/blob/master/gms/client/src/main/java/com/linkedin/dataset/client/Datasets.java#L73

Reply:
In this dataprocesses `autocomplete` method, `requestFilters` is `@Nullable` so check whether it is null needed (similar to this (https://github.com/linkedin/datahub/blob/master/gms/client/src/main/java/com/linkedin/dataset/client/Datasets.java#L116)
for sure, I can change the method signature and make requestFilters` is `@Nonnull`

Reply:
Based on our discussion, I will do a clean up/refactor later of this and other clients.